packages/usdk/packages/upstreet-agent/packages/react-agents/classes/conversation-object.ts

```diff
@@ -97,6 +104,21 @@ export class ConversationObject extends EventTarget {
   getAgent() {
     return this.agent;
   }
+
+  setCurrentAgentPlayer(player: Player) {
```

Comment:
If we unify the data model, these probably won't be needed.

Reply:
removed with update

Reply:
As the Agent's Player is itself available in the agentsMap, any functions related to the currentAgent are removed